tequilapi/client/client.go

```diff
@@ -228,6 +228,24 @@ func (client *Client) Unlock(identity, passphrase string) error {
 	return nil
 }
 
+// Payout registers payout address for identity
+func (client *Client) Payout(identity, ethAddress string) error {
+	path := fmt.Sprintf("identities/%s/payout", identity)
+	payload := struct {
+		EthAddress string `json:"ethAddress"`
+	}{
+		ethAddress,
+	}
+
+	response, err := client.http.Put(path, payload)
+	if err != nil {
+		return err
+	}
+	defer response.Body.Close()
+
+	return nil
```

Comment:
Shouldn't we validate at least HTTP status here?

Reply:
Why it should be validated here?

Reply:
because you can get stuff like 404, 503 and what not from the put response. Not having an error is not an indicator of success here.

Reply:
[`parseResponseError`](https://github.com/mysteriumnetwork/node/blob/master/tequilapi/client/http_client.go#L128) will handle statuses like 404, 503 by creating error